use-hot-module-reload/tsconfig.json

```diff
@@ -3,8 +3,7 @@
   "compilerOptions": {
     "outDir": "lib",
     "strict": true,
-    "esModuleInterop": true,
-    "module": "ES2020",
+    "module": "Preserve",
```

Comment:
`esModuleInterop` is `true` when using `module: preserve`. More details: https://github.com/microsoft/TypeScript/pull/56785